infoschema/tables_test.go

```diff
@@ -1031,29 +1035,32 @@ func (s *testClusterTableSuite) TestSelectClusterTable(c *C) {
 	slowLogFileName := "tidb-slow.log"
 	prepareSlowLogfile(c, slowLogFileName)
 	defer os.Remove(slowLogFileName)
-	tk.MustExec("use information_schema")
-	tk.MustExec("set @@global.tidb_enable_stmt_summary=1")
-	tk.MustQuery("select count(*) from `CLUSTER_SLOW_QUERY`").Check(testkit.Rows("1"))
-	tk.MustQuery("select count(*) from `CLUSTER_PROCESSLIST`").Check(testkit.Rows("1"))
-	tk.MustQuery("select * from `CLUSTER_PROCESSLIST`").Check(testkit.Rows(":10080 1 root 127.0.0.1 <nil> Query 9223372036 0 <nil> 0 "))
-	tk.MustQuery("select query_time, conn_id from `CLUSTER_SLOW_QUERY` order by time limit 1").Check(testkit.Rows("4.895492 6"))
-	tk.MustQuery("select count(*) from `CLUSTER_SLOW_QUERY` group by digest").Check(testkit.Rows("1"))
-	tk.MustQuery("select digest, count(*) from `CLUSTER_SLOW_QUERY` group by digest").Check(testkit.Rows("42a1c8aae6f133e934d4bf0147491709a8812ea05ff8819ec522780fe657b772 1"))
-	tk.MustQuery("select count(*) from `CLUSTER_SLOW_QUERY` where time > now() group by digest").Check(testkit.Rows())
-	tk.MustExec("use performance_schema")
-	re := tk.MustQuery("select * from `CLUSTER_events_statements_summary_by_digest`")
-	c.Assert(re, NotNil)
-	c.Assert(len(re.Rows()) > 0, IsTrue)
-	tk.MustQuery("select * from `CLUSTER_events_statements_summary_by_digest_history`")
-	c.Assert(re, NotNil)
-	c.Assert(len(re.Rows()) > 0, IsTrue)
-	tk.MustExec("set @@global.tidb_enable_stmt_summary=0")
-	re = tk.MustQuery("select * from `CLUSTER_events_statements_summary_by_digest`")
-	c.Assert(re, NotNil)
-	c.Assert(len(re.Rows()) == 0, IsTrue)
-	tk.MustQuery("select * from `CLUSTER_events_statements_summary_by_digest_history`")
-	c.Assert(re, NotNil)
-	c.Assert(len(re.Rows()) == 0, IsTrue)
+	for i := 0; i < 2; i++ {
```

Comment:
Why do this?

Reply:
https://github.com/pingcap/tidb/pull/14344/files#diff-9cb2947f24dfdff2e6f6cec84b53aa56R1040
Test for no stream and with stream.